src/main.cc

```diff
@@ -117,7 +117,11 @@ int daemon_main_(LiteClient& client, uint64_t interval, bool return_on_sleep) {
 
   std::shared_ptr<LiteClient> client_ptr{&client, [](LiteClient* /*unused*/) {}};
   AkliteClientExt akclient{client_ptr, false, true};
-  akclient.CompleteInstallation();
+  auto finalize_result = akclient.CompleteInstallation();
```

Comment:
I would check whether there is pending installation before running `CompleteInstallation`:

```
if (client.IsInstallationInProgress) {
   auto finalize_result = akclient.CompleteInstallation();
  ...
}
```

Reply:
Done.